### PR TITLE
EPMRPP-39187: edit widget - js error

### PR DESCRIPTION
**Pull request by Andrew-Omelchenko**

### CODE REVIEW

**Comment by Andrew-Omelchenko** [comment]
Warning fixed. Also fixed `Section "Save widget on dashboard" should not be visible when editing widget - it should be visible only when adding widget from launches page.` -> bug from EPMRPP-38706.

File: app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/dashboardControl/dashboardControl.jsx [hunk 1]
```diff
@@ -12,44 +13,57 @@ const messages = defineMessages({
...
+
+    return (
+      <div className={cx('dashboardControl')}>
+        <h5 className={cx('dashboardControl-title')}>
+          {formatMessage(messages.dashboardControlTitle)}
+        </h5>
+        <div className={cx('dashboardControl-list')}>
+          {newDashboards.map((dashboard, index) => {
+            const active = value.id ? value.id === dashboard.id : index === 0;
+            const onClick = () => {
```
Comment (ultd39) [inline app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/dashboardControl/dashboardControl.jsx]: This handler pretty simple for create a variable for it - just put it into `onClick` prop: `onClick={() => onChange(dashboard)}`.

Comment (Andrew-Omelchenko) [reply app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/dashboardControl/dashboardControl.jsx]: Fixed

**Comment by Andrew-Omelchenko** [comment]
Fixed

**Pull request merged**